tests/rptest/services/redpanda.py

```diff
@@ -96,6 +96,12 @@
     ),
 ]
 
+# Log errors that are expected in tests that restart nodes mid-test
+CHANGE_REPLICATION_FACTOR_ALLOW_LIST = [
+    re.compile(
+        "cluster - .*Unable to allocate topic with given replication factor"),
```

Comment:
Can you find what's emitting this log, and see if it makes sense to log at WARN instead of ERROR severity? (it's something that is a valid response to user input, rather than an internal error in redpanda)

Reply:
It should not be logged with an error, can you post here a full log line for this error ?